dev-docs/acceptance-testing/domain_services_acceptance_checklist.md

```diff
@@ -20,7 +20,7 @@ settings: (node integrations) a few integrations\ already created, (node credent
 
     ```
     ssh your-ad-user@a2-local-inplace-upgrade-acceptance.cd.chef.co
-    sudo chef-automate admin-token
+    sudo chef-automate iam token create admin-token-1 --admin
     ```
 
     * Or [Use the Automate UI](https://github.com/chef/automate/blob/master/components/compliance-service/smokin/README.md#generating-a-token)
```

Comment:
i think the steps linked to here are gonna need to be updated, to indicate that after the token is created that they need to add it to a policy... idk if we want to pass that over to the super team though?